types/nuxt-i18n.d.ts

```diff
@@ -68,7 +68,7 @@ declare namespace NuxtVueI18n {
       baseUrl?: string | ((context: NuxtContext) => string)
       detectBrowserLanguage?: DetectBrowserLanguageInterface | false
       langDir?: string | null
-      lazy?: boolean
+      lazy?: boolean | {skipNuxtState: boolean}
```

Comment:
```suggestion
      lazy?: boolean | {skipNuxtState?: boolean}
```

The `NuxtVueI18n` namespace is deprecated though and I would suggest switching to `Options` type imported from `nuxt-i18n`. The types are also correct there.

Reply:
@rchl `import {Options} from 'nuxt-i18n'` doesn't find anything. Besides, I'm relying on the augmented type of the Nuxt config which types the `i18n` to `AllOptionsInterface`, so if we're supposed to move to a different type, that augmented type should be updated as well.